### PR TITLE
Magento_Framework: avoid using deprecated escape* methods from Abstra…

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Avoid using deprecated escape* methods from AbstractBlock

### Related Pull Requests

https://github.com/magento/magento2/pull/31610

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: dev/tests/integration/testsuite/Magento/Framework/View/_files/test_template.phtml [hunk 1]
```diff
@@ -13,6 +13,7 @@
  *
  * @var \Magento\Framework\View\Element\BlockInterface $block
  * @var \Magento\Framework\Escaper $escaper
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline dev/tests/integration/testsuite/Magento/Framework/View/_files/test_template.phtml]: It looks like this is not needed

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: dev/tests/integration/testsuite/Magento/Framework/View/_files/test_template.phtml (hunk 1 above)
Comment (engcom-Hotel) [reply dev/tests/integration/testsuite/Magento/Framework/View/_files/test_template.phtml]: Has been removed.

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Review by ihor-sviziev — APPROVED** [review]
✔ Approved.

Failing tests looks not related to changes form this PR.

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests CE, Functional Tests EE, Static Tests

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests CE, Functional Tests EE

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests CE

**Pull request closed without merging**